src/common/POFields/FieldAssignedTo.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'redux-form';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  TextField,
+  IconButton,
+} from '@folio/stripes/components';
+import {
+  withStripes,
+  stripesShape,
+  Pluggable,
+} from '@folio/stripes/core';
+
+const columnMapping = {
+  name: <FormattedMessage id="ui-orders.user.name" />,
+  patronGroup: <FormattedMessage id="ui-orders.user.patronGroup" />,
+  username: <FormattedMessage id="ui-orders.user.username" />,
+  barcode: <FormattedMessage id="ui-orders.user.barcode" />,
+};
+const visibleColumns = ['name', 'patronGroup', 'username', 'barcode'];
+
+const styles = {
```

Comment:
I think we have to use something like CSSinStripes https://github.com/folio-org/stripes-components/blob/master/docs/CSSinStripes.md with separate css files